### PR TITLE
Fix authentication in notebook

### LINKED ISSUE #1021: Notebook '04-context-variables-chat' not working
**Describe the bug**

Just checked out the code, and going through the notebooks in `samples/notebooks/python`, I'm finding that `04-context-variables-chat.ipynb` isn't working. 

**To Reproduce**

On running the authentication cell, I get an error:

---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In[1], line 10
      8 # Configure AI service used by the kernel
      9 if useAzureOpenAI:
---> 10     api_key, endpoint = sk.azure_openai_settings_from_dot_env()
     11     kernel.add_text_completion_service("dv", AzureTextCompletion("text-davinci-003", api_key, endpoint))
     12 else:

ValueError: too many values to unpack (expected 2)

**Expected behavior**

This should work.... like the similar cells in the other notebooks (e.g. 02).... I can see that the API calls are different

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Win11
 - IDE: Jupyter notebooks / Python 3.11
 - NuGet Package Version [e.g. 0.1.0]

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by riedgar-ms**
### Motivation and Context

Fix an authentication issue with the fourth Python notebook, by coping from one of the others.

Fixes #1021 

### Description

Copy/paste from another notebook

### Contribution Checklist

- [ ] The code builds clean without any errors or warnings
- [ ] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [ ] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [ ] All unit tests pass, and I have added new tests where possible
- [ ] I didn't break anyone :smile:

### CODE REVIEW

File: samples/notebooks/python/04-context-variables-chat.ipynb [hunk 1]
```diff
@@ -208,7 +208,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "await chat(\"if I read that book, what exactly will I learn about Greece history?\")"
+    "await chat(\"if I read that book, what exactly will I learn about Greek history?\")"
```
Comment (riedgar-ms) [inline samples/notebooks/python/04-context-variables-chat.ipynb:211]: Extra fix, thrown in for free.

File: samples/notebooks/python/04-context-variables-chat.ipynb [hunk 2]
```diff
@@ -220,6 +220,14 @@
...
     "await chat(\"could you list some more books I could read about this topic?\")"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "70bbadf0",
+   "metadata": {},
+   "outputs": [],
+   "source": []
```
Comment (dluc) [inline samples/notebooks/python/04-context-variables-chat.ipynb:230]:
```suggestion
```

**Review by alexchaomander — APPROVED** [review]
Thanks you for fixing this!

**Comment by riedgar-ms** [comment]
> Thank you for fixing this!

Did I miss a script I should have run to clean up the notebook?

**Review by shawncal — APPROVED** [review]

**Pull request merged**